test/integration.spec.js

```diff
@@ -10,7 +10,7 @@ const webpackPromise = function (config, globalPlugins) {
   const compiler = webpack(config);
   compiler.outputFileSystem = new MemoryFileSystem();
   if (Array.isArray(globalPlugins)) {
-    globalPlugins.forEach(p => compiler.apply(p));
+    globalPlugins.forEach(p => p.apply(compiler));
```

Comment:
why this change ?

Reply:
`compiler.apply` is deprecated

Reply:
does this still works in webpack 2 ? (which is the devDependency)

Reply:
Yes